recipe/meta.yaml

```diff
@@ -25,10 +25,8 @@ requirements:
     - python
     - pickleshare
     - simplegeneric >0.8
-# These are listed in the setup.py, but do not cause the build tests to fail.
-# In order to enable these dependencies, additional tests will need to be defined.
-#    - prompt_toolkit >=0.60
-#    - pygments
+    - prompt_toolkit >=1.0.3,<2.0.0
+    - pygments
```

Comment:
Most of those are only the extras that are legacies of the pre-Jupyter days. The uncondiational platform dependencies: pathlib2, appnope, colorama, and win_unicode_console should be included, though.